dpgen/generator/styles/gromacs.py

```diff
@@ -0,0 +1,130 @@
+import os
+import json
+from typing import List, TYPE_CHECKING, Tuple, Iterator, Dict, Union
+
+import dpdata
+import numpy as np
+from dpgen.generator.lib.utils import create_path, expand_idx
+from dpgen.generator.model_devi import ModelDeviEngien, Trajectory, Frame
+try:
+    from gromacs.fileformats.mdp import MDP
+except ImportError:
+    # JZ: Do not excuse users using other eigiens
+    #dlog.info("GromacsWrapper>=0.8.0 is needed for DP-GEN + Gromacs.")
+    pass
+
+
+@ModelDeviEngien.register("gromacs")
+class GromacsEngien(ModelDeviEngien):
+    def make_input(self, iter_index: int, sys_index: int, directories: Iterator[str], conf_name: str, models: List[str]):
```

Comment:
@AnguseZhang I think the Gromacs's logic here is a bit strange that is different from LAMMPS. LAMMPS can convert any format to `lammps/lmp` through dpdata, but Gromacs just copies directories. Could dpdata support Gromacs as well? Should we keep difference between two engiens here?